src/patternfly/components/Badge/examples/Badge.md

```diff
@@ -41,6 +41,26 @@ cssPrefix: pf-v6-c-badge
 {{/badge}}
 ```
 
+### Disabled
+```hbs
+{{#> badge badge--modifier="pf-m-unread pf-m-disabled"}}
+  7
+  {{#> screen-reader}}unread messages{{/screen-reader}}
+{{/badge}}
+{{#> badge badge--modifier="pf-m-unread pf-m-disabled"}}
+  24
+  {{#> screen-reader}}unread messages{{/screen-reader}}
+{{/badge}}
+{{#> badge badge--modifier="pf-m-read pf-m-disabled"}}
+  240
+  {{#> screen-reader}}unread messages{{/screen-reader}}
+{{/badge}}
+{{#> badge badge--modifier="pf-m-read pf-m-disabled"}}
+  999+
+  {{#> screen-reader}}unread messages{{/screen-reader}}
+{{/badge}}
```

Comment:
This block is good except the read badges have "unread messages" in the sr-only text. Looking at the other examples, we don't include sr-only text for read badges. In looking at the badge hbs, I see we have support for the sr-only text to be added via `badge--IsUnread`, which could be used here. And with `badge--IsDisabled` from this PR and `badge--text`, these could be written inline as:

```suggestion
{{> badge badge--IsUnread=true badge--IsDisabled=true badge--text="7"}}
{{> badge badge--IsUnread=true badge--IsDisabled=true badge--text="24"}}
{{> badge badge--IsDisabled=true badge--text="240"}}
{{> badge badge--IsDisabled=true badge--text="999+"}}
```

Reply:
@mcoker I missed this previously but just pushed this change.  Slight scope creep but I added an `else` to the .hbs template to always add the read or unread class, and updated the other two examples in the .md file to follow your suggestion above for consistency.